src/index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const regression=require('./regression');
+const regression = require('./regression');
```

Comment:
We use AirBnB’s style guide to write consistent code. Check for errors by running:

```shell
npm run lint
```

This is also wired up to git’s pre-commit hook.